app/src/main/java/com/fishhackathon/ghostgear/network/SearchApi.java

```diff
@@ -0,0 +1,67 @@
+package com.fishhackathon.ghostgear.network;
+
+import android.util.Log;
+
+import com.fishhackathon.ghostgear.models.Net;
+import com.fishhackathon.ghostgear.models.NetInput;
+import com.parse.FindCallback;
+import com.parse.ParseException;
+import com.parse.ParseObject;
+import com.parse.ParseQuery;
+
+import java.util.ArrayList;
+import java.util.List;
+
+/**
+ * Search for information about nets.
+ */
+public final class SearchApi {
+    private static final String TAG = SearchApi.class.getSimpleName();
+
+    private static final String NETS_TABLE = "nets";
+
+    // The rows of NETS_TABLE
+    private static final String COLOR = "color";
+    private static final String MESH_SIZE = "meshSize";
+    private static final String NET_CODE = "netCode";
+    private static final String NUMBER_OF_STRANDS = "numberOfStrands";
+    private static final String ORIGIN = "origin";
+    private static final String TWINE_DIAMETER = "twineDiameter";
+
+    // TODO: Add columns for photos and net use.
+
+    private SearchApi() {};
+
+    public interface SearchCallback {
+        public void done(List<Net> nets);
+
+        // TODO: Do we want to provide an error callback?
```

Comment:
yea some sort of string so we can show a dialog that says, sorry! our app is shitty. maybe save for later as a nice to have.